docs/resources/covid-19.md

```diff
@@ -104,99 +212,505 @@ From the response of Safe Place API, you will get an overall safety score and sc
+        "eligiblePerson": "Some travellers",
+        "quarantineType": "Self",
+        "duration": 7,
+        "referenceLink": "http://ncov.mohw.go.kr/shBoardView.do?brdId=2&brdGubun=23&ncvContSeq=6413",
+        "mandateList": "http://ncov.mohw.go.kr/shBoardView.do?brdId=2&brdGubun=23&ncvContSeq=6268",
+        "quarantineOnArrivalAreas": [
+          {
+            "code": "AD",
+            "areaType": "country"
+          },
+
+          ...
+
+          {
+            "code": "ZW",
+            "areaType": "country"
+          }
+        ]
+      }
+```
+
+##### Area access restrictions to area health pass
+
+```json
+    "areaHealthPass": {
+        "lastUpdate": "2022-05-26",
+        "isRequired": "No",
+        "applicationGuidance": "<p>Effective 1 March, authorities suspended a vaccine pass mandate nationwide, with people no longer required to present proof of being fully vaccinated or possess negative test results to enter public places, due to a high vaccination rate and legal challenges. The suspension may later be reinstated if necessary.</p>"
+      }
+```
+
+#### Area policy
```

Comment:
this is a bit bigger "####" than the above examples

Reply:
Yes, this is on purpose, because the above ones are under the Area access restrictions header, which is on the same level as Area policy